README.md

```diff
@@ -57,18 +57,7 @@ Beacon supports both.
       pool_size: 10
     ```
 
-7.  Configure Tailwind:
```

Comment:
We can just inject an empty profile here https://github.com/BeaconCMS/beacon/blob/abb3aaade3adae66212a67b5ef1f2139349c5625/lib/beacon/css_compiler.ex#L20-L21

Eventually we'll support custom tailwind config but that should be good for now.